dist/zlux-file-explorer/README.md

```diff
@@ -0,0 +1,24 @@
+# ZluxFileExplorer
```

Comment:
Thanks for the awesome README.md 👺  @wadkarsatish 

a) can we put this into the original README? It was only until I saw the diff that I noticed this at all

b) can you add the point we talked about clarifying 'sandbox' 1-2 sentences from our DM is perfect